api/routes/api.php

```diff
@@ -33,7 +34,7 @@
     //    // Routes here
     // });
 
-    // Route::group(['middleware' => ['restrictRole:staff']], function () {
-    //     // Routes here
-    // });
+    Route::group(['middleware' => ['restrictRole:staff', 'restrictRole:admin']], function () {
+        Route::resource('staff-orders', OrderController::class);
```

Comment:
no need to add staff, it doesnt do anything with order or regular apis
```suggestion
        Route::resource('orders', OrderController::class);
```

Reply:
This is the update sir.

![image](https://github.com/chevyB/pcbee/assets/89176216/90fc99a5-29c7-4bc2-bdc5-46227a130e55)
![image](https://github.com/chevyB/pcbee/assets/89176216/0cf2eadf-f276-48f4-ab91-00aa7dd429f7)

Reply:
The error is same as this:
https://github.com/chevyB/pcbee?tab=readme-ov-file#be-possible-errors

Reply:
Sir, I already try the solution 
![image](https://github.com/chevyB/pcbee/assets/89176216/565dd07c-28fc-4f35-812b-39f55fc2d504)

Same error 
![image](https://github.com/chevyB/pcbee/assets/89176216/3e675f6b-8062-452d-9e10-407598f58493)

Reply:
This error is has been around for a while already, pls check for solutions here:
https://www.google.com/search?q=laravel+error+storage&oq=laravel+error+storage&gs_lcrp=EgZjaHJvbWUyBggAEEUYOTIGCAEQLhhA0gEINDI0MWowajGoAgCwAgA&sourceid=chrome&ie=UTF-8